### PR TITLE
css-fonts

### PR DESCRIPTION
**Pull request by Lambpato**
Screenshot 2023-01-25 at 12 25 47 PM

### CODE REVIEW

File: css-fonts/styles.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+h1, h2{
+  font-family: 'Libre Barcode 39', cursive;
+  padding-top: 20px;
+  color: lightblue;
+}
+
+p {
+  color: lightgoldenrodyellow;
+  line-height: 35px;
+  }
```
Comment (shawnkost) [inline css-fonts/styles.css:10]: You imported 2 different fonts, but you're only actually using one. You should have one font applied to your headings, and your second font should be applied to the rest of the content or the `p` tag in this case.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Lamberto! One small change below

**Comment by Lambpato** [comment]
sorry about the the font color was low but i did use both

**Comment by shawnkost** [comment]
You're using the browser default font-family for your `<p>` rather than using a font from Google Fonts. You imported 2 fonts, "Libre Barcode 39" & "Yellowtail", but Yellowtail never gets applied anywhere. You need to set a specific font-family on your `<p>` and apply Yellowtail

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**